pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = --ds=osf_tests.settings --tb=short --reuse-db --allow-hosts=127.0.0.1
+addopts = --ds=osf_tests.settings --tb=short --reuse-db --allow-hosts=127.0.0.1,192.168.168.167
```

Comment:
This I'm not quite sure about, but I have to include all the time when debugging, it allows the test app to make requests to it's own container.